packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -84,50 +84,49 @@ - (void)downloadManifestFromURL:(NSURL *)url
                                                      timeoutInterval:EXUpdatesDefaultTimeoutInterval];
   [self _setManifestHTTPHeaderFields:request];
   [self _downloadDataWithRequest:request successBlock:^(NSData *data, NSURLResponse *response) {
+      
+    NSHTTPURLResponse* httpResponse = (NSHTTPURLResponse *)response;
+    NSDictionary* headerDictionary = [httpResponse allHeaderFields];
```

Comment:
```suggestion
    NSDictionary *headerDictionary = [httpResponse allHeaderFields];
```
apply this change across the entire diff

Reply:
would be nice if obj c had linters like TS.